packages/main/src/components/DynamicPageTitle/index.tsx

```diff
@@ -163,6 +171,7 @@ const DynamicPageTitle = forwardRef((props: DynamicPageTitlePropTypes, ref: Ref<
         : titleContainer.borderBoxSize;
       // Safari doesn't implement `borderBoxSize`
       const titleContainerWidth = borderBoxSize?.inlineSize ?? titleContainer.target.getBoundingClientRect().width;
+      setIsPhone(Device.getCurrentRange(titleContainerWidth).name === 'Phone');
```

Comment:
```suggestion
      setIsPhone(Device.getCurrentRange(titleContainerWidth)?.name === 'Phone');
```
Do we need optional chaining here?